python/TestHarness/TestHarness.py

```diff
@@ -127,10 +129,13 @@ def __init__(self, argv, app_name, moose_dir):
                (self.options.cli_args.find('--parallel-mesh') != -1 or self.options.cli_args.find('--distributed-mesh') != -1)):
 
             option_set = set(['ALL', 'PARALLEL'])
-            self.checks['mesh_mode'] = option_set
+            checks['mesh_mode'] = option_set
 
         method = set(['ALL', self.options.method.upper()])
-        self.checks['method'] = method
+        checks['method'] = method
+
+        # This is so we can easily pass checks around to any scheduler plugin
+        self.options._checks = checks
```

Comment:
No? Better way? Because I don't know how to get all this info into the Tester with out passing it around. We could certainly go back to passing three arguments. I don't really care.